### ISSUE TITLE
influx_inspect dumptsi -series does not match series cardinality

### ISSUE THREAD

**Issue by stuartcarnie** [influxdata/influxdb#9482]
Executing the following command should count the number of series, however it appears to report 8x the actual number.

```sh
~/t/c/data3 ❯❯❯ influx_inspect dumptsi -series -series-file=influxdb/data/statsd/_series influxdb/data/statsd | wc -l
 21671400
```

Actual count is about 2.7 million according to `show series cardinality`

**Comment by benbjohnson** [comment]
How many shards do you have?

**Comment by stuartcarnie** [comment]
@benbjohnson there is no tsm data in the `influxdb/data/statsd/...` paths, only 8 partitions in the `_series` path. On there server there were 45 shards.

**Comment by benbjohnson** [comment]
@stuartcarnie I was mistaken. The `dumptsi` command only works on a single shard. However, I added validation and some other fixes in #9489 to prevent usage on non-index directories.
